src/signalo/core/models.py

```diff
@@ -71,3 +99,33 @@ class Sign(ComputedFieldsModel):
     )
     def geom(self):
         return self.geom
+
+
+@receiver(signals.pre_delete, sender=Sign)
+@transaction.atomic()
+def ensure_sign_order_on_delete(sender, instance, *args, **kwargs):
+    """
+    Ensure dense ranking of sign orders despite deletions on same pole
+    """
+    signs_to_update = []
+    for pole in Pole.objects.all():
```

Comment:
same than before, I believe we don't need to iterate over the polls (the poll is referenced by the azimuth which is referenced by the sign, you should be able to use the relations to get the instance)

Reply:
on delete, we should do this operation for only 1 pole, not all the poles